app/Http/Controllers/Halo5/ApiV1Controller.php

```diff
@@ -89,4 +128,4 @@ private function _error($message)
             'message' => $message
         ], 200);
     }
-}
\ No newline at end of file
+}
```

Comment:
yes, blame github editor for doing that. Reported it to them on 4/1/2014 and they said.

> I am not sure it's an empty line that's being inserted. It looks like that because the editor has to move the cursor to the next line, but what's being inserted is the missing line break at the end of the last sentence. If that breaks code in a compiler/interpreter, I am afraid it's the compiler's fault. In any case, I will keep an eye for similar reports and check to see if something is wrong.

> We do have a confirmed problem though. Our diffs don't display this currently, and we have an issue to fix it.

So obviously there either was a disconnected in me explaining the problem or its quite difficult to fix.